.env

```diff
@@ -1,3 +1,12 @@
 EXTEND_ESLINT=true
 
-REACT_APP_API_BASE_URL=http://localhost:3009
\ No newline at end of file
+PORT=8080
+
+# These endpoints can be changed live on the site by updating the window object
+# window.Estuary = { api_endpoint: "", auth_url: "" };
+REACT_APP_API_BASE_URL=https://api.estuary.dev/v1
+REACT_APP_AUTH_BASE_URL=https://auth.estuary.dev
+
+# These settings require an application restart to enable right now
+REACT_APP_SHOW_OIDC_LOGIN=true
+REACT_APP_SHOW_OIDC_LOGIN_GOOGLE=true
```

Comment:
We'll need to eventually think of a system on how we're gonna enable 3rd party stuff in the system better than this.